smt/utils/design_space.py

```diff
@@ -299,7 +299,9 @@ def _decode_dv(x_encoded: np.ndarray, i_dv_decode):
         ]
         return decoded_des_vectors[0] if is_1d else decoded_des_vectors
 
-    def sample_valid_x(self, n: int, unfolded=False) -> Tuple[np.ndarray, np.ndarray]:
+    def sample_valid_x(
+        self, n: int, unfolded=False, random_state=None, **kwargs
```

Comment:
Is `kwargs` really needed? Do we not just use LHS with `criterion="ese"` all the time? Do we need such flexibility at the moment?

Otherwise we should document what is expected in `kwargs`, eg options for the inner LHS sampler

Reply:
Do you want to fix the option to be "ese" and remove kwargs ?

Reply:
Yes! If there is no compelling need, let's simplify!